src/java/org/apache/cassandra/config/Config.java

```diff
@@ -314,6 +314,8 @@ public class Config
     public int otc_coalescing_window_us = otc_coalescing_window_us_default;
     public int otc_coalescing_enough_coalesced_messages = 8;
 
+    public volatile boolean coerce_read_consistency_all = false;
```

Comment:
Could you add a comment describing the type of situations where we would need to set this?